data.sql

```diff
@@ -162,4 +162,8 @@ FULL JOIN specializations ON specializations.vets_id = vets.id
 WHERE vets.name = 'Maisy Smith' 
 GROUP BY S 
 ORDER BY CNT 
-DESC LIMIT 1;
\ No newline at end of file
+DESC LIMIT 1;
+
+--Performance Improvement--
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215530568-b48e1939-eb7a-44a9-b98c-f3aa3451ff58.png)

- [ ] Your MIT link is not working :fearful: It is because you don't have a license file in your repository :thinking: You can add a license file in your repository by following these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: